lua/astrocommunity/pack/biome/init.lua

```diff
@@ -0,0 +1,58 @@
+-- https://biomejs.dev/internals/language-support/
+local supported_ft = {
+  "astro",
+  "css",
+  "graphql",
+  -- "html",
+  "javascript",
+  "javascriptreact",
+  "json",
+  "jsonc",
+  -- "markdown",
+  "svelte",
+  "typescript",
+  "typescriptreact",
+  "vue",
+  -- "yaml",
+}
+
+---@type LazySpec
+return {
+  {
+    "williamboman/mason-lspconfig.nvim",
+    opts = function(_, opts)
+      opts.ensure_installed = require("astrocore").list_insert_unique(opts.ensure_installed, { "biome" })
+    end,
+  },
+  {
+    "WhoIsSethDaniel/mason-tool-installer.nvim",
+    optional = true,
+    opts = function(_, opts)
+      opts.ensure_installed = require("astrocore").list_insert_unique(opts.ensure_installed, { "biome" })
+    end,
+  },
+  {
+    "jay-babu/mason-null-ls.nvim",
+    optional = true,
+    opts = function(_, opts)
+      opts.ensure_installed = require("astrocore").list_insert_unique(opts.ensure_installed, { "biome" })
+    end,
+  },
+  {
+    "stevearc/conform.nvim",
+    optional = true,
+    opts = function(_, opts)
+      if not opts.formatters_by_ft then opts.formatters_by_ft = {} end
+
+      for _, ft in ipairs(supported_ft) do
+        opts.formatters_by_ft[ft] = { "biome" }
+      end
+
+      opts.formatters = opts.formatters or {}
+      opts.formatters.biome = {
+        -- Only use the `biome` formatter if biome config is present
+        require_cwd = true,
```

Comment:
I believe this is undeeded: 
https://github.com/stevearc/conform.nvim/blob/6feb2f28f9a9385e401857b21eeac3c1b66dd628/lua/conform/formatters/biome.lua#L5